pallets/moonbeam-foreign-assets/src/evm.rs

```diff
@@ -318,6 +319,11 @@ impl<T: crate::Config> EvmCaller<T> {
 		)
 		.map_err(|_| Error::<T>::EvmInternalError)?;
 
+		if_std! {
```

Comment:
Not sure if we should use this kind of macros inside the runtime, it might bring problems. 

In my opinion I'd remove it :)

Reply:
I think this is a mistake and should be removed